BiweeklyMeetings/2022-06-28.md

```diff
@@ -0,0 +1,442 @@
+
+**Matthias** -: So I don't know, we can perhaps create a dedicated channel on the CIP server just to avoid people floating the general, and we can keep the general for normal conversations or organizing around the events more than discussing CIPS per se. So I'm not sure how people feel about that. We haven't really planned for this, but I think there is room for having a fast-paced kind of conversation channel for the CIP, which could be just on the server, but not on general.
+
+**M. Àngels** -:  As long as then we are mindful and transfer the important things to GitHub. And I think that is something that each one of us needs to be mindful of. Because if not, it's a lot of overhead for either you the editors or a single person.
+
+**Matthias** -: It is.
+
+**Robert** -: I think it's a great step forward to have a topic-specific channels on Discord. The problem usually is that everyone wants to present their own opinions as if they're consensus. But if you give people a little bit of time to vent their opinions in the more public forum, usually, the consensus emerges on its own. And then one of the participants can copy it back to GitHub at that point, like Matthias just did this morning.
+
+**M. Àngels** -:  I just would like to make sure that it's not always Matthias or any of you that needs to be doing this extra work.
+
+**Matthias** -: And it's also my own perspective what I got from the discussion. So it's obviously, skewed to both my understanding and my own biases. [crosstalk 00:29:09].
+
+**Robert** -: It can go the other way. Also, on the other way, I had been having a review of that thread privately with other correspondence and the author. And I actually should have been more public about posting my conclusions or my summaries, I suppose of the existing discussion back to GitHub. So when we do see common threads emerging, we should be willing to post them. And I think that would clear a lot of it up too.
+
+**Matthias** -: So what I see from the discussion and from the past discussion we had also, and I think I mentioned it already last time that there are multiple things being discussed as part of the CIP and it would be nice to maybe organize the conversation in multiple CIPs or multiple different forms of discussions. Like for instance, the protocol parameters, their value, and possibly changing K and things like that. That's octagonal to the CIP and to just be discussed on the side.
```

Comment:
```suggestion
**Matthias** -: So what I see from the discussion and from the past discussion we had also, and I think I mentioned it already last time that there are multiple things being discussed as part of the CIP and it would be nice to maybe organize the conversation in multiple CIPs or multiple different forms of discussions. Like for instance, the protocol parameters, their value, and possibly changing K and things like that. That's orthogonal to the CIP and to just be discussed on the side.
```